source/includes/rest/_batch_api.md

```diff
@@ -326,3 +326,189 @@ request and the time your data appears in user interface.
+          "provider": "shopify",
+          "upstream_id": "fedcba",
+          "identifier": "Order_654321",
+          "amount": 4900,
+          "tax": 100,
+          "fees": 0,
+          "discount": 0,
+          "permalink": "http://myorders.com/orders/654321",
+          "currency_code": "USD",
+          "properties": {
+            "size": "small",
+            "color": "blue"
+          },
+          "occurred_at": "2013-05-18T10:31:58Z",
+          "closed_at": "2013-05-18T10:35:58Z",
+          "cancelled_at": null,
+          "financial_state": "paid",
+          "fulfillment_state": "fulfilled",
+          "billing_address": {
+            "name": "Bill Billington",
+            "first_name": "Bill",
+            "last_name": "Billington",
+            "company": "Bills R US",
+            "address_1": "123 Bill St.",
+            "address_2": "Apt. B",
+            "city": "Billtown",
+            "state": "CA",
+            "zip": "01234",
+            "country": "United States",
+            "phone": "555-555-5555",
+            "email": "bill@bills.com"
+          },
+          "shipping_address": {
+            "name": "Ship Shippington",
+            "first_name": "Ship",
+            "last_name": "Shipington",
+            "company": "Shipping 4 Less",
+            "address_1": "123 Ship St.",
+            "address_2": "null",
+            "city": "Shipville",
+            "state": "CA",
+            "zip": "01234",
+            "country": "United States",
+            "phone": "555-555-5555",
+            "email": "ship@shipping.com"
+          },
+          "items": [{
+            "id": "8888888",
+            "product_id": "765432",
+            "sku": "4444",
+            "amount": 4900,
+            "name": "Canoe",
+            "quantity": 1,
+            "upstream_id": "hijkl",
+            "upstream_product_id": "opqrs",
+            "upstream_product_variant_id": "zyxwv",
+            "price": 4900,
+            "tax": 100,
+            "fees": 0,
+            "discount": 100,
+            "taxable": true,
+            "properties": {
+              "color": "black"
+            }
+          }]
+        }
+      ]
+    }]
+  }
+  EOF
+```
+
+> Responds with a `202 Accepted` response and an empty JSON response:
+
+```json
+{}
+```
+
+We recommend using this API endpoint when you need to create a collection of orders at once that will likely exceed the regular rate limit of 3,600 requests per hour.
+
+Note: Since our batch APIs process requests in the background, there may be a delay between the time you submit your
+request and the time your data appears in user interface.
+
+### HTTP Endpoint
+
+`POST /:account_id/orders/batches`
+
+### Arguments
+
+<table>
+  <thead>
+    <tr>
+      <th>Key</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>orders</code></td>
+      <td>Required. An Array with between 1 and 1000 objects containing order data.</td>
```

Comment:
We should probably link to the orders section here, so that users can jump to the definition for this data.